v1/proto/service/gribi.proto

```diff
@@ -113,6 +113,13 @@ message ModifyRequest {
   // conidered master, in order to allow for a client to reconnect
   // without an external election having taken place.
   //
+  // Unpon discovering a failover, the server should stop processing pending
+  // AFTOperations that were sent by the previous master. For cases that the
```

Comment:
Should we use 2119-type wording here? It seems this wording is quite loose to me.

Specifically:

 * we're leaving it as an implementation choice as to whether to stop processing the queue of operations from the previous master. Is this a case of "the server MAY stop processing enqueued operations from the previous master" or SHOULD/MUST? It seems like the client needs to be OK with them being processed whatever if this is anything other than a MUST NOT.
 * Should we clarify what happens if the old master has gone away? At that point, we do not expect that the new master gets ACKs for operations that it did not send, such that we'd want this to just be discarded right?

Reply:
Agree. I'll use 2119 wording here. 

* I think the server should stop processing as it would help the new master do the reconciliation (we don't want the pending operation hapenning while tehe acuquiring master is doing Get()).